MultiSource/Benchmarks/Ptrdist/anagram/anagram.c

```diff
@@ -588,7 +594,9 @@ void FindAnagram(_Array_ptr<Quad> pqMask : count(MAX_QUADS),
     ;
 }
 
-int Cdecl CompareFrequency(_Ptr<char> pch1, _Ptr<char> pch2) {
+int Cdecl CompareFrequency(_Ptr<const void> ch1, _Ptr<const void> ch2) {
+  _Ptr<char> pch1 = (_Ptr<char>)ch1;
```

Comment:
What's going on here with the `const` annotations? This seems like an extra change. If need be, cast CompareFrequency to a function type that only uses checked pointers.